paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
+      if(this.inputValue != undefined){
```

Comment:
This checking for undefined is not working. That's the wrong way to check for undefined.    
if(typeof(this.inputValue) != 'undefined' && this.inputValue.trim() != ''){
Also it doesn't make sense to perform the search if the inputValue is empty, because the not candidates will hit by empty string search.